src/bufr/IodaEncoder/IodaEncoder.cpp

```diff
@@ -155,7 +155,11 @@ namespace Ingester
 
 
                 var.atts.add<std::string>("long_name", { varDesc.longName }, {1});
-                var.atts.add<std::string>("units", { varDesc.units }, {1});
+
+                if (!(varDesc.units == "none" || varDesc.units == ""))
```

Comment:
@rmclaren just want you to be aware of this change that @gthompsnJCSDA needs for this PR. For now this change will remain in the feature/sprint-ioda-converters branch which may not get merged into develop for a couple months. If you are okay with this change it might be worthwhile to add it to your feature branch (feature/query ?) and develop.